webapp/channels/src/components/self_hosted_purchases/self_hosted_purchase_modal/success_page.tsx

```diff
@@ -1,58 +0,0 @@
-// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
-// See LICENSE.txt for license information.
-
-import React from 'react';
-import {FormattedMessage} from 'react-intl';
-
-import PaymentSuccessStandardSvg from 'components/common/svg_images_components/payment_success_standard_svg';
```

Comment:
We probably can delete several of the images that are going with these components